### PR TITLE
Development

### PR DESCRIPTION
**Pull request by baconlover2020**
Implementing the critical function described in issue #66

### CODE REVIEW

**Comment by CodingPenguin1** [comment]
I think we should move it out of bot.py, so it can be updated later. Cogs are nice. Not sure which cog makes the most sense. @tux2603 thoughts?

**Comment by tux2603** [comment]
Of our current cogs I'd personally say that ServerManagement would make the most sense, but it might be better to put it in a new cog

File: bot.py [hunk 1]
```diff
@@ -72,6 +72,15 @@ async def on_command_error(ctx, error):
         await log(client, f'{author} attempted to run `{message}` but failed because of an unexpected error: {error}')
 
 
+@client.event
+async def on_message(message):
+    if message.author == client.user:
+        return
+    channel = message.channel
+    if channel.name == 'i-made-a-poppy':
```
Comment (tux2603) [inline bot.py:80]: There was a typo in the original issue, the channel name here should be 'i-made-a-poopy'

**Review by tux2603 — CHANGES_REQUESTED** [review]

File: bot.py (hunk 1 above)
Comment (CodingPenguin1) [reply bot.py:80]: I'd say we make a new cog for listeners

**Comment by CodingPenguin1** [comment]
> Of our current cogs I'd personally say that ServerManagement would make the most sense, but it might be better to put it in a new cog

I'd say we make a new cog for listeners

**Comment by tux2603** [comment]
Agreed. I say we make a new cog in a file called EvenListeners.py and move it there

**Comment by CodingPenguin1** [comment]
Resolved by #70

**Pull request closed without merging**